.github/workflows/e2e-test.yaml

```diff
@@ -26,8 +33,7 @@ jobs:
       fail-fast: false
       matrix:
         cluster:
-          - k3d
-          - kind
+          - ${{ inputs.cluster || 'k3d' }}
```

Comment:
out of curiosity: does `default: k3d` not guarantee that `inputs.cluster` is set?

Reply:
Nope, `default` for `workflow_call`/`workflow_dispatch` affects only the cases when the workflow is triggered by `workflow_call`/`workflow_dispatch`; if the workflow is triggered by schedule or by push, then `inputs` are not available.

Reply:
huh, fair enough - thanks